andela/bp-esa-frontend
# #167486106 Remove page reload on navigation.

#### What does this PR do?
- This PR ensures that links on the navbar when clicked do not reload the page.

#### Description of Task to be completed?
- Ensure that links on the navbar when clicked do not reload the page
- Fix eslint issues.

#### How should this be manually tested?
1. Git clone the repository.
2. Checkout to the `ch-remove-page-reload-on-navbar-links-167486106` branch.
5. Start the server `npm run start` _(or the `yarn` equivalent)_.
6. Clone the backend repository, `https://github.com/andela/bp-esa-backend`.
7. Checkout to the backend `develop` branch.
8. Start the backend server `npm run start:dev` _(or the `yarn` equivalent)_.
9. Login into the ESA application
10. Click the `Dashboard` link in the application Header section.
You should be redirected to the `Dashboard` page without a page refresh.
11. Click the application logo (ESA) on the far left side of the header.
You should be redirected to the `Automations` page without a page refresh.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[167486106](https://www.pivotaltracker.com/story/show/167486106)

#### Screenshots (If applicable)
N/A
